content/blog/esc-projects-environment-tags-launch/index.md

```diff
@@ -63,6 +63,13 @@ We offer numerous ways for you to get started with Projects via the Pulumi Web C
   pulumi/cloud-pe/demo
  ```
 
+### Breaking Changes
+
+All existing environments should continue to work with the introduction of Projects, belonging now to a `default` project. However, when moving existing environments out of the default project you may notice some differences that we have outlined below to be proactive in communicating possible actions you'll need to migrate your environments.
+
+* If your environment was interpolating the `context.currentEnvironment.name` or `context.rootEnvironment.name` from [Contextual information](/docs/esc/environments/#pulumi-contextual-information) this value will now include the project for any environments outside of the `default` project.
+* If you were using any of the [secrets providers with OIDC](#using-secrets-providers-and-oidc) the subject will be either `pulumi:environments:org:<organization name>:env:<project name>/<environment name>` if no `subjectAttributes` are specified or if the `currentEnvironment.name` subject attribute is specified it will now resolve to `currentEnvironment.name:<project name>/<environment name>`. Updating this value will require manual intervention for the secrets provider in the new environment to work.
```

Comment:
```suggestion
* If you were using any of [Pulumi ESC providers](/docs/esc/providers/) with OIDC, the subject will be either `pulumi:environments:org:<organization name>:env:<project name>/<environment name>` if no `subjectAttributes` are specified or if the `currentEnvironment.name` subject attribute is specified it will now resolve to `currentEnvironment.name:<project name>/<environment name>`.
```

Reply:
going to keep as secrets providers as that's how it's referenced on the concepts page